plugin.json

```diff
@@ -59,6 +59,20 @@
                 "help_text": "The webhook secret used by the ServiceNow API calls to Mattermost for sending message response.",
                 "regenerate_help_text": "Regenerates the secret for ServiceNow Virtual Agent Plugin. Regenerating this key invalidates any existing key.",
                 "default": ""
+            },
+            {
+                "key": "ChannelCacheSize",
+                "display_name": "DM Channel Cache Size:",
+                "type": "number",
+                "help_text": "The size of the cache that is used to store DM channel IDs.",
```

Comment:
should we write the unit as we added for the time below?

Reply:
What would be the unit for the this?